packages/story-editor/src/app/story/utils/isEmptyStory.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { shallowEqual } from '@web-stories-wp/react';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../../elements/utils/createPage';
+
+const isEmptyStory = (pages) => {
+  const hasOnlyOnePage = pages.length === 1;
+  if (!hasOnlyOnePage) {
+    return false;
+  }
+
+  const hasNoExtraElements = pages[0].elements.length === 1;
+  if (!hasNoExtraElements) {
+    return false;
+  }
+
+  const hasNoBackgroundMedia = !('resource' in pages[0].elements[0]);
```

Comment:
`!...elements[0].isDefaultBackground` might be more straightforward / readable perhaps.

Reply:
That will work, just the `isDefaultBackground` is very confusing to me, if you just change the color - it's still true 🤔 

![image](https://user-images.githubusercontent.com/21036224/150433227-64baf6bd-40f8-43aa-bd44-ebe4d534b395.png)

Reply:
`isDefaultBackground` is basically the default background element type (meaning a shape). Even though the shape can have props changed, it's still the default shape. If the default element type changes to sth else (like image or video), it's not the default element anymore. So basically the `isDefaultBackground` means that it's the default shape element, regardless of if it this shape props have changed or not.

We're using it kind of everywhere in the code when checking if it's the default shape or media, so might be good to do it here as well for consistency.

Reply:
Alright, I will use that (`isDefaultBackgroundType` wouldn't be so ambiguous).